pkg/ebpf/net_capture.go

```diff
@@ -166,11 +166,9 @@ func (t *Tracee) processNetCapEvent(event *trace.Event) {
 		layer3 := packet.NetworkLayer()
 		layer4 := packet.TransportLayer()
 
-		ipHeaderLength := uint32(0)     // IP header length is dynamic
-		icmpHeaderLength := uint32(8)   // ICMP header length is 8 bytes
-		icmpv6HeaderLength := uint32(4) // ICMPv6 header length is 4 bytes
-		udpHeaderLength := uint32(8)    // UDP header length is 8 bytes
-		tcpHeaderLength := uint32(0)    // TCP header length is dynamic
+		ipHeaderLength := uint32(0)  // IP header length is dynamic
+		udpHeaderLength := uint32(8) // UDP header length is 8 bytes
```

Comment:
If UDP header is fixed, must this be a constant then?

Reply:
nah, I guess its ok, don't want to mix const and vars here.